ajax/libs/downloadjs/package.json

```diff
@@ -17,7 +17,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/rndme/download.js"
+    "url": "https://github.com/rndme/download"
```

Comment:
It's different from the package.json of origin repo.
```
  "repository": {
    "type": "git",
    "url": "https://github.com/rndme/download.git"
  },
```
ref: https://github.com/rndme/download/blob/master/package.json#L19~L22